zbus/src/connection/connect.rs

```diff
@@ -0,0 +1,62 @@
+use std::{future::Future, pin::Pin};
+use tracing::debug;
+
```

Comment:
Oh and you forgot to convert this mod to the `connect/mod.rs`. Can you also check that you didn't forget other modules?

Reply:
ok